tests/test_init.py

```diff
@@ -2,27 +2,184 @@
+
+            The received output of systemd-analzye time is the following:
+
+            Startup finished in 505ms (userspace)
+            graphical.target reached after 453ms in userspace
+
+            For getting the startup time, use prefix = "Startup finished in " and the function will return 0.505s
+            """
+            i = stdout.find(prefix)
+            if i < 0:
+                raise ValueError(f"prefix {prefix} not found")
+            t = (
+                stdout[i + len(prefix) + 1 :].strip().split(" ")[0]
+            )  # take the next word
+            if t.endswith("ms"):
+                return float(t[:-2]) / 1000.0
+            elif t.endswith("s"):
+                return float(t[:-1])
+            else:
+                raise ValueError("time unit not recognized")
+
+        time = auto_container.connection.run_expect(
+            [0], "systemd-analyze time"
+        )
+        startup = extract_time(time.stdout, "Startup finished in ")
+        assert startup <= STARTUP_LIMIT, "Startup threshold exceeded"
+        target = extract_time(time.stdout, ".target reached after ")
+        assert (
+            target <= TARGET_LIMIT
+        ), "Reaching systemd target threshold exceeded"
+
+    def test_systemd_nofailed_units(self, auto_container):
+        """
+        Ensure there are no failed systemd units
+        """
+        output = auto_container.connection.run_expect(
+            [0], "systemctl list-units --state=failed"
+        )
+        assert (
+            "0 loaded units listed" in output.stdout
+        ), "failed systemd units detected"
+
+    def test_systemd_detect_virt(
+        self, auto_container, host, container_runtime
+    ):
+        """
+        Check that :command:`systemd-detect-virt` correctly detects the container runtime
+        """
+        output = auto_container.connection.run_expect(
+            [0], "systemd-detect-virt"
+        ).stdout
+        runtime = container_runtime.runner_binary
+        assert (
+            runtime in output
+        ), f"systemd-detect-virt failed to detect {runtime}"
+
+    def test_journald(self, auto_container):
+        """
+        Check that :command:`journald` behaves correctly
+        """
+
+        # Check that we reached at least the multiuser target
+        journal = auto_container.connection.run_expect(
+            [0], "journalctl --boot"
+        )
+        assert (
+            "Reached target Multi-User System" in journal.stdout
+        ), "Multi-User target was not reached"
+
+    def test_hostnamectl(self, auto_container, host, container_runtime):
+        """
+        Check expected values from :command:`hostnamectl`
+        """
+
+        hostnamectl = auto_container.connection.run_expect([0], "hostnamectl")
+        # Process the printed values to a string map
+        values = TestSystemd._split_values(hostnamectl.stdout)
+        assert values["Chassis"] == "container", "Chassis mismatch"
+        assert (
```

Comment:
ok, I have just tested `registry.opensuse.org/devel/bci/sle-15-sp3/images/bci/bci-init:latest` and it also contains ` Operating System: SUSE Linux Enterprise Server 15 SP3`, which makes sense, as they are based on sle.

Reply:
I guess this means your comment has been addressed?

Reply:
Yes, but I'm not sure if we should make it more generic and just match `suse` word only instead of full string with `enterprise...`. 
I don't know how future BCI containers will look like in OBS, if they will be "openSUSE" based or what... @dcermak do you know?

Reply:
Ok, fair enough. I'm just too cautious :) 
Let's fix it when it breaks something (if it does).